app/helpers/distribution_helper.rb

```diff
@@ -17,4 +17,12 @@ def hashed_calendar_path
     crypt = ActiveSupport::MessageEncryptor.new(Rails.application.secret_key_base[0..31])
     distributions_calendar_url(hash: crypt.encrypt_and_sign(current_organization.id))
   end
+
+  def quantity_by_item_id(distribution, item_id)
+    item_id = Integer(item_id)
```

Comment:
```suggestion
    item_id = item_id.to_i
```
Is this functionally the same? If so, I've got a subjective prefence to use this one unless good reason to use `Integer`

Reply:
I am 100% fine with doing `to_i` -- I've gotten in the habit of using `Integer(...)` at my job for cases like these.

The only notable difference is one of coercion:
![Screen Shot 2022-10-31 at 9 22 39 AM](https://user-images.githubusercontent.com/502363/199017834-20e9d4a4-d660-4a4b-b7f2-269bfc5c6ab8.png)

Execution fails loudly with `Integer` and quietly with `to_i`. That's really the only difference to choose one over the other.